packages/search-in-workspace/src/browser/search-in-workspace-result-tree-widget.tsx

```diff
@@ -187,7 +189,8 @@ export class SearchInWorkspaceResultTreeWidget extends TreeWidget {
                     }
                 } else {
                     const children: SearchInWorkspaceResultLineNode[] = [];
-                    const icon = await this.labelProvider.getIcon(new URI(result.file));
+                    const fileStat = await this.fileSystem.getFileStat(result.file);
+                    const icon = (fileStat) ? await this.labelProvider.getIcon(fileStat) : undefined;
```

Comment:
Nit: unnecessary parentheses arounf `fileStat`?

Reply:
Just wondering, what is displayed if `icon` ends up undefined?

Reply:
> Just wondering, what is displayed if `icon` ends up undefined?

if the icon is `undefined` then the following occurs, it acts as a regular tree node:
<img width="337" alt="screen shot 2018-11-22 at 6 46 27 pm" src="https://user-images.githubusercontent.com/40359487/48924495-f64bef00-ee86-11e8-8a7a-1042d5cd2126.png">

Reply:
Ok so just no icon, makes sense :)